### DISCUSSION TITLE
Can we have this theme for Android Studio?

### DISCUSSION THREAD

**Discussion by gilbertohnrq** [primer/github-vscode-theme#310] [Ideas]
I've been using this theme on vscode since I found out about it and I really love the colors, but now I'm working with Android Studio and I can't find any theme that looks like this one on their marketplace, could we have this theme for Android Studio aswell?

**Comment by mellson** [comment]
@gilbertohnrq you can find a `GitHub Dark` variant in [Material UI Lite](https://plugins.jetbrains.com/plugin/12124-material-theme-ui-lite). But I agree a native version would be great

**Comment by anhnt** [comment]
Please make a native theme for IntelliJ as well :)

**Comment by simurai** [comment]
> @anhnt Please make a native theme for IntelliJ as well :)

See this comment https://github.com/primer/github-vscode-theme/issues/26#issuecomment-660939835 with a link to a port.

**Comment by simurai** [comment]
@gilbertohnrq Related https://github.com/primer/github-vscode-theme/discussions/151#discussioncomment-638695
